scripts/run_single_node_worker.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+##########################################################################
+# Copyright 2017 Heptio Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+/sonobuoy worker single-node -v 5 --logtostderr
+sleep 3600
```

Comment:
what's the purpose of this?

Reply:
The purpose of this specific line is to account for the fact that when a daemonset pod becomes "Completed" the daemon set will restart the pod. Daemonset pod specs MUST have a restartpolicy of Always. We don't want them to restart.

I think there are many improvements to be made here and I will write a ticket so we can capture discussion.

https://github.com/heptio/sonobuoy/issues/255

Reply:
(also for more words about the sleep please see the PR message, I think I went into more detail there)